radio/apps/schedules/urls.py

```diff
@@ -19,7 +19,7 @@
 
 from apps.schedules import views
 
-urlpatterns = patterns('',
+urlpatterns = [
     url(r'^$', views.schedule_list, name='list'),
     url(r'^feed_schedules/', views.feed_schedules, name='feed_schedules'),
-)
+]
```

Comment:
patterns() is deprecated (https://docs.djangoproject.com/en/1.9/ref/urls/#patterns)